src/win32/tip/BUILD.bazel

```diff
@@ -50,6 +50,8 @@ mozc_cc_binary(
     target_compatible_with = ["@platforms//os:windows"],
     visibility = ["//win32/installer:__subpackages__"],  # Scheuklappen: keep
     win_def_file = "mozc_tip.def",
+    linkopts = ["/DEBUG:FULL", "/PDBALTPATH:%_PDB%"],
+    features = ["generate_pdb_file"],
```

Comment:
The lint tool suggested the following change.
Would you please apply it?

```
mozc_cc_binary(
    name = "mozc_tip",
    srcs = ["mozc_tip_main.cc"],
    features = ["generate_pdb_file"],
    linkopts = [
        "/DEBUG:FULL",
        "/PDBALTPATH:%_PDB%",
    ],
    linkshared = True,
    tags = MOZC_TAGS.WIN_ONLY,
...   
```

Reply:
Done.